src/exo/LoopIR_scheduling.py

```diff
@@ -1169,6 +1169,22 @@ def map_e(self, e):
             return super().map_e(e)
 
 
+def get_reads(e):
```

Comment:
Is this redundant with the other PR, #236?

Reply:
Yeah it's the exact same function